packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -113,19 +273,11 @@ export = {
 
     const node = mesh.addVirtualNode('test-node', {
       dnsHostName: 'test.domain.local',
-      listener: {},
     });
 
     const stack2 = new cdk.Stack();
 
-    const node2 = appmesh.VirtualNode.fromVirtualNodeName(stack2, 'imported-node', mesh.meshName, node.virtualNodeName);
-
-    node2.addListeners({
-      portMapping: {
-        port: 8081,
-        protocol: appmesh.Protocol.TCP,
-      },
-    });
+    appmesh.VirtualNode.fromVirtualNodeName(stack2, 'imported-node', mesh.meshName, node.virtualNodeName);
```

Comment:
Sorry, why these changes here...?

Reply:
[Reference](https://github.com/aws/aws-cdk/pull/10793#discussion_r518413076)
We have moved the `addListener` function from `IVirtualNode` to `VirtualNode`. So we cannot use addListener function to add a listener to an imported node.